app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
+        return
```

Comment:
Ensure that the input string contains exactly three elements separated by spaces, with `mv` as the first element. This is crucial to prevent errors when accessing parts of the command.